sdk/communication/azure-communication-phonenumbers/CHANGELOG.md

```diff
@@ -1,14 +1,20 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (2025-06-20)
 
 ### Features Added
+- GA release of the reservations functionality.
+- API version `2025-04-01` is the default.
```

Comment:
Not sure if "API version `2025-04-01` is the default." should be here, considering it was part of the previous beta release. Also, I think it should mention that purchases from countries requiring a Do Not Resell Agreement is also GA now.